build/merge.py

```diff
@@ -1,9 +1,5 @@
 """
-Run this with PWD being the top level of the repo and
-python build/merge.py
-
-This script will join the relevant columns from the TX Comptroller data file
-(texas_corp_merged.h5) with the cleaned Bexar property data (bexar_property_all.h5)
+Run this with PWD being the top level of the repo and python build/merge.py
```

Comment:
this docstring should also include a summary of what this code does (e.g. what you deleted)